package.json

```diff
@@ -18,54 +18,57 @@
     "schemas/"
   ],
   "dependencies": {
-    "@hapi/boom": "^9.1.0",
-    "@hapi/hapi": "^19.1.1",
-    "@hapi/vision": "^6.0.0",
-    "@microfleet/core": "^17.3.0",
-    "@microfleet/transport-amqp": "^15.0.0",
-    "@microfleet/validation": "^9.0.0",
+    "@babel/parser": "7.22.0",
+    "@hapi/boom": "^10.0.1",
+    "@hapi/hapi": "^21.3.2",
+    "@hapi/vision": "^7.0.1",
+    "@microfleet/core": "^19.2.6",
+    "@microfleet/transport-amqp": "^17.4.0",
+    "@microfleet/validation": "^11.1.3",
     "bunyan": "^1.8.1",
     "bunyan-noop": "^2.0.0",
     "bunyan-sentry-stream": "^1.1.0",
-    "chalk": "^4.0.0",
+    "chalk": "^5.2.0",
     "common-errors": "^1.0.0",
-    "glob": "^7.0.6",
+    "glob": "^10.2.6",
     "is": "^3.1.0",
     "jaeger-client": "^3.13.0",
     "lodash.merge": "^4.6.1",
     "messagebird": "^4.0.1",
-    "ms-conf": "^7.0.0",
-    "prom-client": "^11.2.1",
+    "ms-conf": "^8.1.2",
+    "prom-client": "^14.2.0",
     "raven": "^2.6.4",
-    "stdout-stream": "^1.4.1",
-    "twilio": "^3.24.0"
+    "stdout-stream": "^2.0.0",
+    "twilio": "^4.11.1"
   },
   "devDependencies": {
     "@babel/cli": "^7.1.5",
```

Comment:
could probably kill all babel related stuff, anything we've used has long been released in node

Reply:
adjust compile script (remove it) + change `main` entry once thats done